sysutils/tmux/Portfile

```diff
@@ -83,6 +86,4 @@ post-destroot {
     xinstall -m 0644 ${filespath}/ftdetect-tmux.vim ${destroot}${prefix}/share/vim/vimfiles/ftdetect/tmux.vim
 }
 
-notes "If you want integration with system pasteboard consider installing port tmux-pasteboard as well"
-
```

Comment:
did you intend to remove the notes? It does appear to me that this is unrelated to the commit message, and the notes look still relevant... or?

Reply:
The trac issue linked is addressed by the commit as well. The [homepage of tmux-pasteboard](https://github.com/ChrisJohnsen/tmux-MacOSX-pasteboard) states that the project isn't needed on newer versions of tmux.
> You may not even need the program from this repository! tmux version 2.6 (2017 September) incorporated the functionality of the wrapper program. If you are using a new enough tmux, you may be able to skip using the wrapper in your tmux configuration.

The note can only address individuals in legacy systems, if possible. That's a good compromise.